app/src/main/java/com/codesoom/assignment/ErrorCreate.java

```diff
@@ -0,0 +1,9 @@
+package com.codesoom.assignment;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+// BAD_REQUEST에서 NOT_FOUND로 수정 -> 차이점 파악할 것
```

Comment:
이런 주석은 다음과 같이 `TODO` 태그를 달아 줍니다. 프로젝트 전체에 퍼져 있는 할 일들을 한번에 찾기 쉬워져요.
 
```suggestion
// TODO: BAD_REQUEST에서 NOT_FOUND로 수정 -> 차이점 파악할 것
```